internal/runners/activate/activate.go

```diff
@@ -116,6 +124,32 @@ func (r *Activate) run(params *ActivateParams, activatorLoop activationLoopFunc)
 	return activatorLoop(r.out, r.subshell, projectPath, activate)
 }
 
+func updateProject(prjFile *projectfile.Project, names *project.Namespaced) (*project.Project, error) {
+	if names.CommitID == nil || *names.CommitID == "" {
+		latestID, fail := model.LatestCommitID(names.Owner, names.Project)
+		if fail != nil {
+			return nil, locale.WrapInputError(fail.ToError(), "err_set_namespace_retrieve_commit", "Could not retrieve the latest commit for the specified project {{.V0}}.", names.String())
+		}
+		names.CommitID = latestID
```

Comment:
Feels a little dirty and error-prone using the namespace reference to record the commitID that you want to end up using. Suggest using a local var for this instead.